app/views/comments/index.html.erb

```diff
@@ -1,7 +1,7 @@
 <% if @root_comment.present? %>
-  <% title @root_comment.title + " — DEV" %>
+  <% title("#{@root_comment.title} - #{ApplicationConfig["COMMUNITY_NAME"]}") %>
```

Comment:
Should we not use the `community_name` helper for these cases? Being able to change/override how this behaves by changing a single method seems desirable.

Reply:
Yes we should, thanks! I think it was late, and I'm not sure I missed these 🤦🏽‍♀️